### PR TITLE
Add zeroconf discovery support

### PR DESCRIPTION
**Pull request by pawlizio**
Add zeroconf discovery support for Velux KLF200 devices on the network.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.10", "3.11", "3.12"]
+        python-version: ["3.11", "3.12"]
```
Comment (Julius2342) [inline .github/workflows/ci.yml:16]: ^^

**Comment by Julius2342** [comment]
Shouldnt we integrate this discovery in the pyvlx module itself?

**Comment by pawlizio** [comment]
I'm not sure, don't we need a separate pyvlx instances for every KLF200 which we find on the network or are we able to manage them inside one pyvlx instance? My idea for this PR was just to use it on HA side when someone start a manual configuration flow for VELUX in the UI, so that we can use zeroconf to check the network and create a list of identified KLF200 devices on the network. Those flows are used before an instance of pyvlx is created. 
Basically HA supports natively a zeroconf flow step as well. I already considered this in my HA core PR [106660](https://github.com/home-assistant/core/pull/106660)  [here](https://github.com/pawlizio/core/blob/0f0765d20d71f6407b5807bf32ace8b1baa3e19d/homeassistant/components/velux/config_flow.py#L77).
